modules/node/cms_node_api.e

```diff
@@ -279,6 +299,18 @@ feature -- Change: Node
 			node_storage.update_node (a_node)
 		end
 
+
+feature -- Node status
+
+	Not_published: INTEGER = 1
+			-- The node is not published.
+
+	Published: INTEGER = 2
+			-- The node is published.
+
+	Trashed: INTEGER = 3
```

Comment:
Maybe we could use  -1

not_published: 0   (as default)
published: 1
trashed: -1

-1 sound a standard status to say ... trashed.

to be discussed.